gradlew

```diff
@@ -1,25 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Where did the new gradle wrapper script come from? Are there any benefits of migrating?

Reply:
Huh. TIL. I've never seen that command before – I normally either let Studio do the upgrade or manually go into the `gradle/wrapper/gradle-wrapper.properties` file and do it manually.

Let's keep these changes then.